src/services/api.tsx

```diff
@@ -56,6 +56,7 @@ const useGetUrlsQuery = ({ enabled = true }: { enabled?: boolean }) => {
     return useQuery({
         queryKey: ['urls'],
         enabled: enabled,
+        refetchOnWindowFocus: false,
```

Comment:
- can we move this to the global config (queryClient in _app.tsx) so that other get APIs have the same functionality?

Reply:
Done